shards/modules/gfx/shader/core_shards.hpp

```diff
@@ -812,6 +887,20 @@ struct WithInput {
   static SHTypesInfo inputTypes() { return CoreInfo::NoneType; }
   static SHTypesInfo outputTypes() { return CoreInfo::NoneType; }
 
+  static SHOptionalString help() {
+    return SHCCSTR(
+        "This shard creates a conditional statement within a shader code. If the vertex attribute (or the interpolated value of the vertex attribute supplied to the pixel) specified in the Name parameter is being received at the stage that calls this shard, "
```

Comment:
I would avoid terms like "interpolated value of the vertex attribute", vertex outputs can be anything they don't necessarily have to be based on vertex attributes, although often are

Reply:
Just calling them vertex stage outputs seems enough to me, as long as it's clarified in detail elsewhere